lcls-twincat-motion/Library/DUTs/ST_AxisParameterSetExposed.TcDUT

```diff
@@ -0,0 +1,76 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<TcPlcObject Version="1.1.0.1">
+  <DUT Name="ST_AxisParameterSetExposed" Id="{ee296551-543a-4bbb-98b7-123ff2229439}">
+    <Declaration><![CDATA[TYPE ST_AxisParameterSetExposed :
+    // Collects the axis parameters that are intentionally exposed to EPICS.
+STRUCT
+    // Maximum Dynamics
+    {attribute 'pytmc' := '
+        pv: MaxVel
+        io: i
+        field: DESC Maximum commandable speed of the axis in EU/s.
+    '}
+    fVeloMaximum                : LREAL; // Maximum commandable speed of the axis in EU/s.
```

Comment:
I thought these needed to be above the parameter but I was wrong, these associate properly in the IDE as-is